i18n/README.id.md

```diff
@@ -0,0 +1,91 @@
+<p align="center">
+<img width="300" src="https://raw.githubusercontent.com/supabase/supabase/master/web/static/supabase-light-with-background.svg"/>
+</p>
+
+---
+
+# Supabase
+
+[Supabase](https://supabase.io) adalah sumber terbuka alternatif Firebase. Kita mengembangkan fitur Firebase menggunakan alat sumber terbuka kelas perusahaan.
+
+- [x] Menggunakan database Postgres yang di host
+- [x] Langganan waktu sebenarnya
+- [x] Otentikasi dan otorisasi
+- [x] API yang dibuat secara otomatis
+- [x] Dashboard
+- [x] Penyimpanan
+- [ ] Functions (segera hadir)
+
+## Dokumentasi
+
+Untuk dokumentasi lengkap, kunjungi [supabase.io/docs](https://supabase.io/docs)
+
+## Komunitas & Bantuan
+
+- [Forum Komunitas](https://github.com/supabase/supabase/discussions). Terbaik utnuk: membantu dengan mengembangkan, diskusi tentang praktik terbaik database.
+- [GitHub Issues](https://github.com/supabase/supabase/issues). Terbaik untuk: bug dan kesalahan yang kamu temui saat menggunakan Supabase.
+- [Email Support](https://supabase.io/docs/support#business-support). Terbaik untuk: masalah - masalah dengan database atau infrastrukturkamu
+
+## Status
+
+- [x] Alpha: Kita test Supabase dengan sejumlah pelanggan tertutup.
```

Comment:
This sounds too stiff, you can improve it by replacing `test` with `menguji`. Also I think it's better to add `secara` between `pelanggan` and `tertutup`.

`Kami menguji Supabase dengan sejumlah pelanggan secara tertutup` sounds a bit better.